cognite/powerops/custom_modules/power_model_v1/data_models/containers/SHOPResult.container.yaml

```diff
@@ -0,0 +1,100 @@
+space: '{{powerops_models}}'
+externalId: SHOPResult
+name: SHOPResult
+usedFor: node
+properties:
+  timeseries:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: timeseries
+    description: Timeseries from output from shop
+  tsDirect1:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirect1
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirect2:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirect2
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirect3:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirect3
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirectList1:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirectList1
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirectList2:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirectList2
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  preRun:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: preRun
+    description: The prerun file from the shop run
+  shopMessages:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: shopMessages
+    description: The prerun file from the shop run
+  cplexLogs:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: cplexLogs
+    description: The cplex logs from the shop run
+  postRun:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: postRun
+    description: The post run file from the shop run
+  scenario:
```

Comment:
I think we should point to the `case` instead (and then we can find the `scenario` from the `case`)

Reply:
Ok, yes makes sense since it includes the start and end times